testing-unit/src/main/java/com/bumble/appyx/testing/unit/helper/ParentNodeTestHelper.kt

```diff
@@ -0,0 +1,38 @@
+package com.bumble.appyx.testing.unit.helper
+
+import androidx.lifecycle.Lifecycle
+import com.bumble.appyx.core.children.nodeOrNull
+import com.bumble.appyx.core.node.ParentNode
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
+
+fun <Routing : Any, N : ParentNode<Routing>> N.parentNodeTestHelper() =
+    ParentNodeTestHelper(this)
+
+class ParentNodeTestHelper<Routing : Any, N : ParentNode<Routing>>(
+    private val node: N
+) : NodeTestHelper<N>(
+    node = node
+) {
+
+    fun <Routing : Any> assertChildHasLifecycle(routing: Routing, state: Lifecycle.State) {
+        val childMap = node.children.value
+        val key = childMap.keys.find { it.routing == routing }
+
+        if (key != null) {
+            childMap[key]?.nodeOrNull?.also {
```

Comment:
If `nodeOrNull` returns null, perhaps that should also fail? we should definitely make sure that an assertion function doesn't fail silently.

Reply:
done